source/renderer/app/i18n/locales/en-US.json

```diff
@@ -357,6 +357,13 @@
   "settings.wallets.currency.titleLabel": "Display ada balances in other currency",
   "sidebar.category.settings.notification.label": "New discreet mode settings are available in <b>Security</b> tab.",
   "sidebar.wallets.addWallet": "Add wallet",
+  "sidebar.wallets.search.placeholder": "Filter",
+  "sidebar.wallets.sortByBalanceButton": "Balance",
+  "sidebar.wallets.sortByBalanceTooltip": "Sort by balance",
+  "sidebar.wallets.sortByDateButton": "Date",
+  "sidebar.wallets.sortByDateTooltip": "Sort by creation date",
+  "sidebar.wallets.sortByNameButton": "A-Z",
```

Comment:
what type of dash is here? "-" / "–" / "—" ? 

<img width="45" alt="image" src="https://user-images.githubusercontent.com/2539345/145386068-1a0f5977-11bf-4c43-8996-9147d8e10ac1.png">

it feels too small

Reply:
Changed to `A – Z`